bigtable/admin/table_admin.cc

```diff
@@ -22,9 +23,45 @@ namespace btproto = ::google::bigtable::admin::v2;
 
 namespace bigtable {
 inline namespace BIGTABLE_CLIENT_NS {
-std::string TableAdmin::CreateInstanceName() const {
-  return absl::StrCat("projects/", client_->project(), "/instances/",
-                      instance_id_);
+::google::bigtable::admin::v2::Table TableAdmin::CreateTable(
+    std::string table_id, std::map<std::string, GcRule> column_families,
+    std::vector<std::string> splits,
+    ::google::bigtable::admin::v2::Table::TimestampGranularity granularity) {
```

Comment:
We only support millisecond granularity right now. At the very least we should document this, or perhaps not expose it at all.

Reply:
Also, thoughts on creating a `TableConfig` or similar structure to make it easier to add new table params in the future?

Reply:
I think we should expose the API to set the granularity because if we don't then when we do it is a backwards incompatible change that would require a new major version number.  Though maybe it would require that anyway.

Not opposed to `TableConfig` and in this case probably makes sense.